website/templates/profile/oauth_app_detail.mako

```diff
@@ -0,0 +1,110 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Application Detail</%def>
+<%def name="content()">
+<style type="text/css">
+    .border-box {
```

Comment:
New class (see screenshot): box with top and bottom borders (no sides). Didn't see an existing class like this, but UI subject to revision.

<img width="698" alt="screen shot 2015-07-13 at 10 56 23 am" src="https://cloud.githubusercontent.com/assets/2957073/8652680/fb005f9c-294d-11e5-9933-eff3dadd1b6a.png">